docs/topics/cache.txt

```diff
@@ -293,6 +293,11 @@ above example, if your server runs as the user ``apache``, make sure the
 directory ``/var/tmp/django_cache`` exists and is readable and writable by the
 user ``apache``.
 
+.. warning::
+    **If the caches LOCATION is a sub directory of** :setting:`MEDIA_ROOT`, :setting:`STATIC_ROOT` **or** :setting:`STATICFILES_FINDERS`, **this can lead to arbitrary remote code execution or information leaking.**
```

Comment:
possessive: `cache's`
`subdirectory`
also note the rest of the file wraps at approx. 79 chars